parsers/julia.c

```diff
@@ -872,7 +872,7 @@ static void deInitLexer (lexerState *lexer)
 static void debugLexer (lexerState *lexer)
 {
     printf("Current lexer state: line %d, token (%lu), cur char `%c`, token str:\n\t`", lexer->line, lexer->cur_token, lexer->cur_c);
-    printf(lexer->token_str->buffer);
+    printf(vStringValue(lexer->token_str));
```

Comment:
This is dangerous code.
This should be
```
printf("%s", vStringValue(lexer->token_str));
```
If the value for ` vStringValue(lexer->token_str)` is %s, un expected address is accessed to fill the value for `%s`.